example/hello_world.py

```diff
@@ -81,10 +81,21 @@ def dimensions(test):
 
 def attachments(test):
   test.Attach('test_attachment', 'This is test attachment data.')
-  with tempfile.NamedTemporaryFile() as f:
-    f.write('This is a file attachment')
-    f.flush()
-    test.AttachFromFile(f.name)
+  # Windows doesn't allow reading from an open NamedTemporaryFile, so we
+  # have to do this the hard way.  Typically in a real test, you would have
+  # invoked some external program that generated an output file.
```

Comment:
what about using delete=False, then replacing f.flush() with f.close()? might make this less crazy?